nx.json

```diff
@@ -0,0 +1,12 @@
+{
+  "namedInputs": {
+    "tiny-product": [
+      "{projectRoot}/**/*",
+      "!{projectRoot}/node_modules/**/*"
+    ],
+    "tiny-library": [
+      "{projectRoot}/**/*",
+      "!{projectRoot}/node_modules/**/*"
+    ]
+  }
```

Comment:
Keeping these in sync with the premium nx build could be a pain. I suppose eventually we're going to combine the two and then we can simplify the open source build away from lerna/nx.